test/SmokeTest.php

```diff
@@ -425,26 +425,32 @@ public function requestProvider() : Traversable
                 'application/vnd.elife.subject+json; version=1',
             ];
         }
-        yield $path = '/search?type[]=reviewed-preprint' => [
-            $this->createRequest($path, 'application/vnd.elife.search+json; version=2'),
-            'application/vnd.elife.search+json; version=2'
-        ];
-        yield $path = '/search' => [
-            $this->createRequest($path, 'application/vnd.elife.search+json; version=1'),
-            'application/vnd.elife.search+json; version=1'
-        ];
-        yield $path = '/search?for=cell' => [
-            $this->createRequest($path, 'application/vnd.elife.search+json; version=1'),
-            'application/vnd.elife.search+json; version=1',
-        ];
-        yield $path = '/search?subject[]=cell-biology' => [
-            $this->createRequest($path, 'application/vnd.elife.search+json; version=1'),
-            'application/vnd.elife.search+json; version=1',
-        ];
-        yield $path = '/search?start-date=2017-01-01&end-date=2017-01-01' => [
-            $this->createRequest($path, 'application/vnd.elife.search+json; version=1'),
-            'application/vnd.elife.search+json; version=1',
-        ];
+
+        foreach ([2, 1] as $version) {
+            yield $path = '/search?type[]=reviewed-preprint' => [
+                $this->createRequest($path, 'application/vnd.elife.search+json; version='.$version),
+                'application/vnd.elife.search+json; version='.$version,
+                200,
+                [],
+                1 === $version ? 0 : null,
```

Comment:
This means that although `type[]` can be anything it should return no results if version is 1.

Reply:
why it should return `null` on version 2?

Reply:
null is how we don’t perform the check. Awe only need to ensure that no results are return for version 1.